pkg/controllers/nodeclaim/consistency/termination.go

```diff
@@ -48,7 +48,7 @@ func (t *Termination) Check(ctx context.Context, node *v1.Node, nodeClaim *v1bet
 	if err != nil {
 		return nil, err
 	}
-	pods, err := nodeutils.GetNodePods(ctx, t.kubeClient, node)
+	pods, err := nodeutils.GetPods(ctx, t.kubeClient, node)
```

Comment:
Why can't this be IsActive()? We use the same criteria in the pdblimits code.

Reply:
I'm mainly trying to keep this consistent with the disruption code which grabs all pods and then passes the full set of pods into the PDB evaluation function